elasticsearch.tf

```diff
@@ -31,11 +31,42 @@ resource "aws_cloudwatch_log_resource_policy" "es_cloudwatch_policy" {
   policy_name     = "dmarc-import-es-cloudwatch-policy"
 }
 
+# Policy document that allows authenticated Cognito users access to the
+# Elasticsearch domain
+data "aws_iam_policy_document" "es_cognito_auth" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "es:*",
```

Comment:
Are we ok with wildcard permissions? Just verifying because of some such as `es:DeleteDomain`.

Reply:
I'm ok with the wildcard here, but I will create an issue for posterity in case someone ever wants to clamp it down.

Reply:
Issue https://github.com/cisagov/dmarc-import-tf-module/issues/50 has been created for this.